index.html

```diff
@@ -0,0 +1,75 @@
+        </div>
+        <hr>
+ 
+        <div id="tribute-info">
+ 
+            <!--Achievements and other
+                details of the person-->
+            <h1 class="title-Boseman">
+                About the Legend
+            </h1>
+
+            <p>
+                ☛ Chadwick Aaron Boseman (/ˈboʊzmən/;[5] November 29, 1976[a] – August 28, 2020) was an American actor. Throughout his career, Boseman received various accolades, including two Screen Actors Guild Awards, a Golden Globe Award, a Critics' Choice Movie Award, four MTV Movie & TV Awards, and nominations for an Academy Award, a Primetime Emmy Award and a BAFTA Award..<br><br>
+                ☛ After studying directing at Howard University, he began his career in theatre, winning a Drama League Directing Fellowship and an acting AUDELCO, along with receiving a Jeff Award nomination for his 2005 play Deep Azure.<br><br>
+                ☛ Transitioning to the screen, his first major role was as a series regular on the NBC drama Persons Unknown (2010) and he landed his breakthrough performance in 2013 as baseball player Jackie Robinson in the biographical film 42. He continued to portray historical figures, starring as singer James Brown in Get on Up (2014) and as attorney Thurgood Marshall in Marshall (2017)..<br><br>
+                ☛Boseman achieved international fame for playing the superhero Black Panther in the Marvel Cinematic Universe (MCU) from 2016 to 2019. He appeared in four MCU films, including an eponymous 2018 film that earned him an NAACP Image Award for Outstanding Actor in a Motion Picture and a Screen Actors Guild Award for Outstanding Performance by a Cast in a Motion Picture. As the first black actor to headline an MCU film, he was also named in the 2018 Time 100.
+                <br><br> ☛He will be for ever greatly remembered for his role in marvel series where he acted as  <strong>
+                “The BlackPanther.”</strong> am movie that greatly sold out arond all cinemas in the world
+                <br><br> ☛ In 2016, Boseman was diagnosed with colon cancer. He kept his condition private, continuing to act until his death in 2020 from the illness. His final film, Ma Rainey's Black Bottom, was released posthumously the same year to critical acclaim, earning him an Academy Award nomination for Best Actor and a Golden Globe Award for Best Actor – Motion Picture Drama. Boseman also received four nominations at the 27th Screen Actors Guild Awards for his work in Da 5 Bloods and Ma Rainey's Black Bottom, the most for a performer at a single ceremony,[6] winning Male Actor in a Leading Role for the latter.
+                <br><br> ☛ Boseman died at his Los Angeles home as a result of complications related to colon cancer on August 28, 2020, with his wife and family by his side.[159][160][161] He was 43 years old.
+                <br>
+            </p>
+ 
+ 
+        </div>
+  
+        <br>
+            For more information,
+        check 
```

Comment:
Your information bar is not under any type of tag, please use a p tag or an h6 tag.